rust/fastsim-core/src/simdrive/simdrive_impl.rs

```diff
@@ -895,13 +886,9 @@ impl RustSimDrive {
         self.cyc_tire_inertia_kw[i] = (0.5
             * self.veh.wheel_inertia_kg_m2
             * self.veh.num_wheels
-            * self.cyc_whl_rad_per_sec[i].powf(2.0)
-            / self.cyc.dt_s_at_i(i)
-            - 0.5
-                * self.veh.wheel_inertia_kg_m2
-                * self.veh.num_wheels
-                * (self.mps_ach[i - 1] / self.veh.wheel_radius_m).powf(2.0)
-                / self.cyc.dt_s_at_i(i))
+            * (self.cyc_whl_rad_per_sec[i].powi(2)
+                - (self.mps_ach[i - 1] / self.veh.wheel_radius_m).powi(2))
+            / self.cyc.dt_s_at_i(i))
```

Comment:
@calbaker These seems like different behavior, could you please elaborate for me?

Reply:
I refactored it from the form `a*b - a * c` to the form `a * (b - c)` to make use of transitive property of multiplication to reduce the number of multiply/divide operations and arguably make the code more readable.

Reply:
I hadn't seen lines 887 and 888! All good